docs/content/v0.20/release.md

```diff
@@ -0,0 +1,60 @@
+# Making KubeStellar Releases
+
+This document defines how releases of the KubeStellar repository are made.
```

Comment:
do we want to create a release of KS repos at the same time and with the same identifier? 
like it is done in kubernetes?

I think that's a little confusing when user needs to use different versions.
for example: 
KS controllers version `v0.2.1`, kubeflex version `v0.5.0`, ocm-status-addon `0.4.0` and ocm-transport-plugin version `v0.1.0`.

instead I would expect that all versions should be aligned. than we create release notes for a specific version - 
for example - 
when releasing v0.6.0 let's assume we put in the release notes that status conflict is solved. 
if users want to use KS version where status conflict is solved, one should use v0.6.0 for KS controllers and for status-addon (kubeflex is a different story cause it's a space management framework, not really related to KS).

Reply:
I'm not sure this is needed.  The fact that it's in the same ORG is not critical. I see the other repos as external dependencies similar to KIND, OCM CLI, etc..  
The main thing is to avoid circular dependency.  Other than that it's a regular A depends on B, and it's a common procedure how we update A to use a new version of B.

Reply:
I would definitely leave KubeFlex to have its independent release cycle as it is going to be decoupled anyway. As for the other repos, I thought that once you define the interface for a "pluggable transport" that transport adapter (which I assume will include status in the future) can evolve independently as long as the interface stays the same. What about other possible transports in the future? Do they all have to release with the same release cycle? It seems to defeat the purpose of having created a pluggable transport in the first place.

Reply:
Normally separate repos are versioned separately.

Reply:
you are all correct. I haven’t thought about it to the end.
no need to align versions.